pwndbg/commands/elf.py

```diff
@@ -17,6 +19,15 @@
 def elfsections() -> None:
     local_path = pwndbg.aglib.file.get_proc_exe_file()
 
+    # IF PIE is set and binary is in execution, save relocation base address.
+    pie_relocation = False
+    if pwndbg.aglib.proc.alive and ELF(local_path).pie:
+        for mapping in pwndbg.aglib.vmmap.get():
+            if mapping.objfile == local_path:
+                pie_relocation = True
+                base_addr = mapping.start
+                break
```

Comment:
Please change this to:
```py
base_address = next((p.start for p in pwndbg.aglib.vmmap.get() if p.objfile == pwndbg.aglib.proc.exe), 0)
```

And then do: `if base_address != 0:` - show relocated.